### PR TITLE
[13.0][ADD] l10n_es: new 0% spanish taxes BOE-A-2022-22685 Real Decreto-ley…

### PR DESCRIPTION
**Pull request by acysos**
… 20/2022, de 27 de diciembre

Hello,

This PR add the spanish taxes aproved at 27th december. BOE: https://www.boe.es/buscar/doc.php?id=BOE-A-2022-22685

This new taxes are very urgent, because this taxes come in effect at  1st January 2023.

@jco-odoo can you check this.

Greetings

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Same comment as in https://github.com/odoo/odoo/pull/108855#issuecomment-1367963190. I will start with this one.

File: addons/l10n_es/data/account_data.xml [hunk 1]
```diff
@@ -69,5 +69,8 @@
         <record id="tax_group_iva_5" model="account.tax.group">
             <field name="name">IVA 5%</field>
         </record>
+        <record id="tax_group_iva_0" model="account.tax.group">
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_data.xml:72]: Already present in https://github.com/odoo/odoo/pull/108846/files#diff-be65fc06bf41c02157a65fa726a0c8e3cbc7c477eafa7fc9dc0fb87ae5d76d9fR6

File: addons/l10n_es/data/account_tax_data.xml [hunk 2]
```diff
@@ -1609,6 +1609,44 @@
...
+        <field name="name">0% IVA soportado (Bienes)</field>
+        <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="tax_group_id" ref="tax_group_iva_0"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+                'tag_ids': [ref('mod_303_28')],
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_tax_data.xml:1624]: They shouldn't have tags, as they will be another that are still unknown.

File: addons/l10n_es/data/account_tax_data.xml [hunk 3]
```diff
@@ -1609,6 +1609,44 @@
             }),
         ]"/>
     </record>
+    <record id="account_tax_template_p_iva0_bc" model="account.tax.template">
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_tax_data.xml:1612]: This XML-ID is already used: https://github.com/odoo/odoo/blob/a597c54b5257e6388437e41527826ec8108f564e/addons/l10n_es/data/account_tax_data.xml#L2304

**Comment by pedrobaeza** [comment]
With your permission, superseded by #108868, as it needs hurry.

**Review by rafaelbn — COMMENTED** [review]
👍 Thank you!

**Pull request closed without merging**